ci/meta.yml

```diff
@@ -24,26 +24,27 @@ meta:
         outputs:
           - name: terraform-config
       params:
-        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
-        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
         AWS_REGION: eu-west-2
-    terraform-apply:
-      task: terraform-apply
+
+    terraform-common-config:
       config:
         platform: linux
         image_resource:
           type: docker-image
           source:
-            repository: ((terraform.repository))
-            version: ((terraform12.version))
-            tag: ((terraform12.version))
+            repository: hashicorp/terraform
```

Comment:
is the repo not in secretes manager?

Reply:
No idea, just following aws-metrics-infrastructure and aws-management-infrastructure

Reply:
yes it is. `((dataworks.terraform_repository))` But it doesn't feel like it should be a hard requirement

Reply:
Yea, I was also copying previous ones, but maybe it _should_ be using the secrets one if it's there, same with mine.

Reply:
it enables us to move all references quickly when Docker Hub is not working

Reply:
Yea, agreed. I've asked @liamraemclauchlan to update it. He copied it from a source we hadn't updated.

Reply:
Potentially, but as it's a non-private/secret parameter, I don't have any feelings either way.  
As a side bar, if you want to know how to quest SM to get the list of stored params, give me a prod on Slack.